internal/liqonet/network-manager/tunnelendpointcreator/tunnelendpointcreator.go

```diff
@@ -382,10 +384,10 @@ func (tec *TunnelEndpointCreator) createTunnelEndpoint(ctx context.Context, para
 	// here we create it
 	tep := &netv1alpha1.TunnelEndpoint{
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: tunEndpointNamePrefix,
+			GenerateName: fmt.Sprintf("tun-endpoint-%s-", param.remoteCluster.ClusterName),
```

Comment:
Since the resources are of type tun-endpoint maybe it is a bit repetitive to call them tun-endpoint?